app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "network-api",
+  "description": "API and CMS for The Mozilla Network site",
+  "addons": [
+    "heroku-postgresql"
+  ],
+  "scripts": {
+    "postdeploy": "python app/manage.py migrate"
+  }
```

Comment:
yeah but to do that you need to specify all the environment vars here according to the heroku docs iirc